### DISCUSSION TITLE
The remote server is not a valid

### DISCUSSION THREAD

**Discussion by klemenNeco** [saran-surya/email_auth_node#5] [Q&A]
Hi guys!

I followed all the steps until the end, but when I deployed the project and linked it to flutter, the output said: 

`email-auth >> Initialising Email-Auth server
email-auth >> Initialising Email-Auth server
--- Package Error ---
Unable to access remote server. 😑
--- End Package Error ---

--- package Error ---

email-auth >> The remote server is not a valid.`

I am really not that good at flutter so I have no idea what to do.

**Comment by saran-surya** [comment]
Hi @klemenNeco , can you please attach the full screenshot of the error, because I am not able to see the full error message.

Also kindly check of you are passing the correct signature for the remoteserverconfig var.

**Comment by saran-surya** [comment]
Hi @klemenNeco Is your issue resolved ?

**Reply by saran-surya** [reply]
Closing this discussion due to inactivity
